src/main/kotlin/racing/RacingCarGarage.kt

```diff
@@ -0,0 +1,26 @@
+package racing
+
+import racing.domain.RacingCarsFactory
+import racing.model.Car
+import racing.model.CarCount
+
+class RacingCarGarage(
+    private val racingCarsFactory: RacingCarsFactory,
+) {
+    private val _cars: MutableList<Car> by lazy { mutableListOf() }
+    val cars: List<Car>
+        get() = _cars
```

Comment:
mutable 객체를 외부에 노출되지 않도록 잘 해주셨네요! 👍 
 하지만 이런 경우에도, 다운캐스팅을 통해 외부에서 cars를 변경할 가능성이 있으니, 
 방어적복사를 추가해주시면, 좀더 안전한 코드를 작성할수 있습니다!
```suggestion
        get() = _cars.toList()
```